### PR TITLE
Add missing params

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/11135

### CODE REVIEW

**Comment by yogkumgit** [comment]
@tseward

**Comment by get-itips** [comment]
> @JohanFreelancer9

@yogkumgit The PR is still draft.

File: skype/skype-ps/skype/Grant-CsExternalAccessPolicy.md [hunk 1]
```diff
@@ -239,6 +254,51 @@ Accept pipeline input: False
...
+### -Rank
+The rank of the policy assignment, relative to other group policy assignments for the same policy type.
+
+```yaml
+Type: Int32
+Parameter Sets: GrantToGroup
+Aliases:
+
+Required: True
+Position: Con nombre
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsExternalAccessPolicy.md:266]:
```suggestion
Position: Named
```

File: skype/skype-ps/skype/Grant-CsTeamsAppPermissionPolicy.md [hunk 2]
```diff
@@ -166,6 +175,36 @@ Accept pipeline input: False
...
+### -Rank
+The rank of the policy assignment, relative to other group policy assignments for the same policy type.
+
+```yaml
+Type: Int32
+Parameter Sets: GrantToGroup
+Aliases:
+
+Required: True
+Position: Con nombre
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsAppPermissionPolicy.md:187]:
```suggestion
Position: Named

```

File: skype/skype-ps/skype/Grant-CsTeamsAppSetupPolicy.md [hunk 3]
```diff
@@ -171,6 +179,36 @@ Accept pipeline input: False
...
+### -Rank
+The rank of the policy assignment, relative to other group policy assignments for the same policy type.
+
+```yaml
+Type: Int32
+Parameter Sets: GrantToGroup
+Aliases:
+
+Required: True
+Position: Con nombre
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsAppSetupPolicy.md:206]:
```suggestion
Position: Named
```

File: skype/skype-ps/skype/Grant-CsTeamsUpgradePolicy.md [hunk 4]
```diff
@@ -235,6 +250,102 @@ Accept pipeline input: False
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -MigrateMeetingsToTeams
+Not supported anymore, see Description section.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsUpgradePolicy.md:286]:
```suggestion
Not supported anymore, see the Description section.
```

File: skype/skype-ps/skype/Grant-CsTeamsVideoInteropServicePolicy.md [hunk 5]
```diff
@@ -213,6 +220,36 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
 For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsVideoInteropServicePolicy.md:255]:
```suggestion
For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Grant-CsTeamsMeetingTemplatePermissionPolicy.md [hunk 6]
```diff
@@ -128,6 +134,21 @@ Accept pipeline input: False
...
+### -Rank
+The rank of the policy assignment, relative to other group policy assignments for the same policy type.
+
+```yaml
+Type: Int32
+Parameter Sets: GrantToGroup
+Aliases:
+
+Required: True
+Position: Con nombre
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsMeetingTemplatePermissionPolicy.md:146]:
```suggestion
Position: Named
```

File: teams/teams-ps/teams/Grant-CsTeamsShiftsPolicy.md [hunk 7]
```diff
@@ -60,6 +76,51 @@ Accept pipeline input: False
...
+### -Rank
+The rank of the policy assignment, relative to other group policy assignments for the same policy type.
+
+```yaml
+Type: Int32
+Parameter Sets: GrantToGroup
+Aliases:
+
+Required: True
+Position: Con nombre
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsShiftsPolicy.md:88]:
```suggestion
Position: Named

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by msbemba — APPROVED** [review]

**Pull request merged**